docs/_includes/usage/cf-typography/usage.md

```diff
@@ -299,8 +298,28 @@ For more information, email
 </a>.
 ```
 
+### Block link
+
+The block link class is used as a mixin in the [jump link](#jump-link) and
+[link list](#link-list-modifier) patterns to convert a standard inline link to
+a block-level element with `padding`, `background`, and `border` at extra-small
+screen sizes. It is not intended to be used directly.
+
+<a class="a-link__block" href="#">
```

Comment:
Should be `a-link a-link__block` if we're following the pattern elsewhere, like jump links below.

Reply:
I wonder if we should remove this example and just let people look at the jump link example.

Reply:
Yeah, seems like it should follow like "Jump link with large link modifier" as "Jump link with block link modifier"